python/nx-cugraph/nx_cugraph/algorithms/link_analysis/hits_alg.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import networkx as nx
+import numpy as np
+import pylibcugraph as plc
+
+from nx_cugraph.convert import _to_graph
+from nx_cugraph.utils import _get_float_dtype, networkx_algorithm
+
+__all__ = ["hits"]
+
+
+@networkx_algorithm(
+    extra_params={
+        'weight : string or None, optional (default="weight")': (
+            "The edge attribute to use as the edge weight."
+        )
+    }
+)
+def hits(G, max_iter=100, tol=1.0e-8, nstart=None, normalized=True, *, weight="weight"):
```

Comment:
XREF: https://github.com/networkx/networkx/pull/7081

`weight=` argument may be added to networkx soon.